lib/atom.hpp

```diff
@@ -88,62 +83,83 @@ class Atom
   int maxit_state = 100;
 
   bool write_debug =
-      false; // If true, files will be printed to store details of the process
+    false; // If true, files will be printed to store details of the process
 
   // Radius models
   double static sphereNuclearModel(int Z, int A);
 
-protected:
+ protected:
   // Fundamental properties
   int Z, A;                  // Nuclear charge and mass number
   double m, M, mu;           // Mass of the orbiting particle (e.g. muon, electron), of
-                             // the nucleus, and effective mass of the system
+  // the nucleus, and effective mass of the system
   double R;                  // Nuclear radius
-  NuclearRadiusModel rmodel; // Nuclear model
-  // Grid
+  NuclearRadiusModel rmodel; // Nuclear model Grid
   double rc = 1.0;   // Central radius
-  double dx = 0.005; // Step
-  // Potential
+  double dx = 0.005; // Step Potential
   CoulombSpherePotential *V_coulomb;
   // Additional potential terms
   bool use_uehling = false;
   UehlingSpherePotential V_uehling;
   bool use_econf = false;
   EConfPotential V_econf;
```

Comment:
Here I think it should stay as:
```cpp
  NuclearRadiusModel rmodel; // Nuclear model
  // Grid
  double rc = 1.0;   // Central radius
  double dx = 0.005; // Step
  // Potential
  double dx = 0.005; // Step Potential
  CoulombSpherePotential *V_coulomb;
  // Additional potential terms
  bool use_uehling = false;
  UehlingSpherePotential V_uehling;
  bool use_econf = false;
  EConfPotential V_econf;
```

Or maybe more clearly as:

```cpp
  NuclearRadiusModel rmodel; // Nuclear model

  // Grid
  double rc = 1.0;   // Central radius
  double dx = 0.005; // Step

  // Potential
  double dx = 0.005; // Step Potential
  CoulombSpherePotential *V_coulomb;
  // Additional potential terms
  bool use_uehling = false;
  UehlingSpherePotential V_uehling;
  bool use_econf = false;
  EConfPotential V_econf;
```

Since `rc` and `dc` are parameters which define the `// Grid`, they should come under that heading. Likewise, `// Potential` here is a heading for `V_coulomb` and `// Additional potential terms`. In other places the long comments overrun the line end but here I think there's a separation between inline comments for individual variables and "section" headings for variables that are grouped by meaning.

Reply:
myself misinterpreted the comments.